src/chains.jl

```diff
@@ -47,7 +47,7 @@ function Chains(val::AbstractArray{A,3},
     axvals = [
         Base.range(start, step=thin, length=size(val, 1)),
         parameter_names,
-        map(i->Symbol("Chain$i"), 1:size(val, 3)),
+        map(i->i, 1:size(val, 3)),
```

Comment:
Shouldn't that be `Symbol.(1:size(val, 3))` or am I missing something?

Reply:
Well, there's no real need to have chain names be symbols. I was lazy and left the `map` call in. I've pushed a modification to make this just `collect(1:size(val, 3))` instead.

Reply:
great!